os/drivers/audio/alc5658.c

```diff
@@ -1563,8 +1570,12 @@ FAR struct audio_lowerhalf_s *alc5658_initialize(FAR struct i2c_dev_s *i2c, FAR
 	priv->lower = lower;
 	priv->i2c = i2c;
 	priv->i2s = i2s;
+#ifndef CONFIG_AUDIO_EXCLUDE_GAIN
```

Comment:
Please make policy first. If you decided return default value instead of error when getVolume, here should be changed.

Reply:
Also this set default value, I think even if it disabled, this should be called.
@btheosam let me know your opinion.

Reply:
If GAIN & VOLUME are disabled, then these two elements of priv structure wont be present. So this is correct.